presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -98,6 +98,8 @@ class PrestoServer {
   virtual std::vector<std::string> registerConnectors(
       const fs::path& configDirectoryPath);
 
+  virtual void registerFileSinks() {}
```

Comment:
Can we add some detailed method description for this method? Not sure what this is for. Thanks!

Reply:
Updated. Thanks!